### PR TITLE
Drop Python 2.7 support

### PR DESCRIPTION
**Pull request by markusdregi**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -59,12 +59,10 @@ def _load_readme():
         "Topic :: Software Development :: Documentation",
         "Topic :: Utilities",
     ],
-    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
+    python_requires=">=3.5",
     install_requires=[
-        "enum34 >= 1.0 ; python_version<'3.4'",
         "six >= 1.0",
```
Comment (ManInFez) [inline setup.py:64]: do you still need six?

Comment (markusdregi) [reply setup.py:64]: Nooo, I definitively do not. Apparently my rebasing went wrong 🤔 Will fix immediately!

**Review by ManInFez — APPROVED** [review]
Huzzah! Remember to update the master version in  komodo-releases :)

**Review by jondequinor — APPROVED** [review]
LGTM

**Pull request merged**